h2o-py/tests/testdir_munging/pyunit_6987_merge_one_empty.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+import random
+
+def mergeOneEmptyFrame():
+    # PUBDEV-6987: merge with one empty frame and one normal frame.
+    file1 = h2o.import_file(pyunit_utils.locate("smalldata/jira/pubdev-6987-a.csv"))
```

Comment:
I believe we don't need it. Let's create a pandas data frame `df = pd.DataFrame({"A1": [1], "A2": [0]})` and use `h2o.H2OFrame(df)`.